web/packages/teleport/src/features.tsx

```diff
@@ -322,6 +328,44 @@ export class FeatureAuthConnectors implements TeleportFeature {
   };
 }
 
+export class FeatureLocks implements TeleportFeature {
+  category = NavigationCategory.Management;
+  section = ManagementSection.Access;
+
+  route = {
+    title: 'Session & Identity Locks',
```

Comment:
Let's see what @roraback thinks, but I'm inclined to just say "Locks" here, since "session" is not something you can actually lock.